NodeManager.cpp

```diff
@@ -1318,6 +1399,113 @@ void SensorBH1750::onReceive(const MyMessage & message) {
 #endif
 
 /*
+   SensorTSL2561
+*/
+#if MODULE_TSL2561 == 1
+// contructor
+SensorTSL2561::SensorTSL2561(int child_id, TSL2561* tsl, int tsl_gain, int tsl_timing, int tsl_spectrum): Sensor(child_id,A5) {
```

Comment:
I'd remove from the constructor the gain, timing and spectrum since will not be used by registerSensor(). But I'd set default values in NodeManager.h